examples/basic/src/main/java/com/soundcloud/lightcycle/sample/basic/BaseActivity.java

```diff
@@ -0,0 +1,12 @@
+package com.soundcloud.lightcycle.sample.basic;
+
+import android.support.v7.app.AppCompatActivity;
+
+import com.soundcloud.lightcycle.ActivityLightCycle;
+import com.soundcloud.lightcycle.LightCycleBaseClass;
+import com.soundcloud.lightcycle.LightCycleDispatcher;
+
+@LightCycleBaseClass
+public abstract class BaseActivity extends AppCompatActivity
+        implements LightCycleDispatcher<ActivityLightCycle<AppCompatActivity>> {
```

Comment:
Do you think we can have a way to specify the Host's the of the dispatcher? 

()https://github.com/soundcloud/lightcycle/issues/48)
> Host : the instance of the Activity or Fragment's lifecycle as seen by the LightCycle.
> The host can be an Activity, a Fragment or anything else (can be an interface like MyView).

Examples :

```
public abstract class BaseActivity extends AppCompatActivity
         implements LightCycleDispatcher<ActivityLightCycle<BaseActivity>> {
```

```
public abstract class BaseActivity extends AppCompatActivity
          implements MyView, LightCycleDispatcher<ActivityLightCycle<MyView>> {
```